tests/test_assets.py

```diff
@@ -1227,117 +1236,28 @@ def test_root_symbols(self):
         as expected.
         """
         # Make sure this successfully gets the chain for CL.
-        cl = FutureChain(self.asset_finder, lambda: '2005-12-01', 'CL')
+        cl = self._get_future_chain('2005-12-01', 'CL')
         self.assertEqual(cl.root_symbol, 'CL')
 
         # These root symbols don't exist, so RootSymbolNotFound should
         # be raised immediately.
         with self.assertRaises(RootSymbolNotFound):
-            FutureChain(self.asset_finder, lambda: '2005-12-01', 'CLZ')
+            self._get_future_chain('2005-12-01', 'CLZ')
 
         with self.assertRaises(RootSymbolNotFound):
-            FutureChain(self.asset_finder, lambda: '2005-12-01', '')
+            self._get_future_chain('2005-12-01', '')
 
     def test_repr(self):
         """ Test the __repr__ method of FutureChain.
         """
-        cl = FutureChain(self.asset_finder, lambda: '2005-12-01', 'CL')
-        cl_feb = FutureChain(self.asset_finder, lambda: '2005-12-01', 'CL',
-                             as_of_date=pd.Timestamp('2006-02-01', tz='UTC'))
+        cl = self._get_future_chain('2005-12-01', 'CL')
 
-        # The default chain should not include the as of date.
-        self.assertEqual(repr(cl), "FutureChain(root_symbol='CL')")
-
-        # An explicit as of date should show up in the repr.
         self.assertEqual(
-            repr(cl_feb),
+            repr(cl),
             ("FutureChain(root_symbol='CL', "
-             "as_of_date='2006-02-01 00:00:00+00:00')")
-        )
-
-    def test_as_of(self):
-        """ Test the as_of method of FutureChain.
-        """
-        cl = FutureChain(self.asset_finder, lambda: '2005-12-01', 'CL')
-
-        # Test that the as_of_date is set correctly to the future
-        feb = pd.Timestamp('2006-02-01', tz='UTC')
-        cl_feb = cl.as_of(feb)
-        self.assertEqual(
-            cl_feb.as_of_date,
-            pd.Timestamp(feb, tz='UTC')
-        )
-
-        # Test that the as_of_date is set correctly to the past, with
-        # args of str, datetime.datetime, and pd.Timestamp.
-        feb_prev = pd.Timestamp('2005-02-01', tz='UTC')
-        cl_feb_prev = cl.as_of(feb_prev)
-        self.assertEqual(
-            cl_feb_prev.as_of_date,
-            pd.Timestamp(feb_prev, tz='UTC')
-        )
-
-        feb_prev = pd.Timestamp(datetime(year=2005, month=2, day=1), tz='UTC')
-        cl_feb_prev = cl.as_of(feb_prev)
-        self.assertEqual(
-            cl_feb_prev.as_of_date,
-            pd.Timestamp(feb_prev, tz='UTC')
-        )
-
-        feb_prev = pd.Timestamp('2005-02-01', tz='UTC')
-        cl_feb_prev = cl.as_of(feb_prev)
-        self.assertEqual(
-            cl_feb_prev.as_of_date,
-            pd.Timestamp(feb_prev, tz='UTC')
+             "as_of_date='2005-12-01 00:00:00+00:00')")
```

Comment:
The quotes and zeros seems like they just add noise to the repr here.  Thoughts on making this just:

`FutureChain('CL', 2005-12-01)`